src/test/java/org/apache/commons/lang3/time/DateUtilsTest.java

```diff
@@ -693,6 +693,59 @@ public void testToCalendar() {
             // expected
         }
     }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNotNull() {
```

Comment:
This test is redundant because we have `testToCalendarWithDateAndTimeZoneNotNull()`